model-engine/model_engine_server/infra/gateways/resources/k8s_endpoint_resource_delegate.py

```diff
@@ -1634,9 +2205,27 @@ async def _get_all_resources(
                 infra_states[key] = (is_key_an_endpoint_id, infra_state)
             except Exception:
                 logger.exception(f"Error parsing deployment {name}")
+        for name, lws_config in leader_worker_sets_by_name.items():
+            # name.startswith("launch-endpoint-id-") should always be true, the other case is a legacy.
+            key = _k8s_resource_group_name_to_endpoint_id(name)
+            is_key_an_endpoint_id = True
+            endpoint_id = key
+            deployment_name = name
+            endpoint_type = (
+                ModelEndpointType.STREAMING
+            )  # TODO change if we ever support other endpoint types
+            infra_states[key] = (
+                is_key_an_endpoint_id,
+                await self._get_resources_from_lws_type(
+                    endpoint_id, deployment_name, endpoint_type, lws_config, all_config_maps
+                ),
+            )
         return infra_states
 
     async def _delete_resources_async(self, endpoint_id: str, deployment_name: str) -> bool:
+
+        # TODO multinode if we ever decide to support that
```

Comment:
what do you mean by this? it seems you added delete_lws here? what else needs to be added for multinode support?

Reply:
Currently multinode async endpoints are not supported, since we don't actually have that template generated. I'd guess that we only need to do the `delete_lws()` so _delete_resources_async probably would work, but I'd want to only mark it as done once async endpoints have been e2e implemented